src/features/team-format-page.js

```diff
@@ -0,0 +1,132 @@
+var debug = require("../debug"),
+    vars = require("../vars"),
+    loadTeam = require("./team-load-team");
+
+module.exports = function () {
+
+    if(bttv.settings.get("formatTeamPage") !== true || $("body").attr("data-page") != "teams#show") return;
+    debug.log("Formatting team page");
+
+    var jquiCSS = $("<link>", {"href":"//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/themes/dark-hive/jquery-ui.min.css", "type":"text/css", "rel":"stylesheet"}),
+        jquiJS  = $("<script>", {"src":"//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js", "type":"text/javascript"}),
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    
+    if(bttv.settings.get("darkenedMode") === true) {
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page-dark.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    }
+    $('body').append(jquiCSS, jquiJS, teamCSS);
+
+    $("h2").remove();
+    $("#about").prepend($("#banner_header"));
+
+    var followTeamButton = $("<a>", {"id":"followTeamBtn", "data-ember-action":"135", "class":"js-follow follow button primary action"});
```

Comment:
``data-ember-action`` should not included.

I'm not sure why, bet I've tended to use camelCase for IDs, and split-with-dashes for classes in BetterTTV.

Please also prefix any id inserted into Twitch's DOM with ``bttv`` to prevent any sort of namespace issues.
For classes, prefix with ``bttv-``.